dandiapi/api/views/private_s3_permissions.py

```diff
@@ -27,7 +27,7 @@ def construct_neuroglancer_url(source, layer_name):
     json_object = {
         "layers": [
             {
-                "type": "new",
+                "type": "image",
```

Comment:
Hi @aaronkanzer, it looks like this change didn't get deployed into production.

Reply:
@kabilar -- thanks for the reminder to manually deploy the API -- should be updated here 🚀